sdk/iotcentral/arm-iotcentral/src/iotCentralClientContext.ts

```diff
@@ -51,10 +62,10 @@ export class IotCentralClientContext extends msRestAzure.AzureServiceClient {
     this.credentials = credentials;
     this.subscriptionId = subscriptionId;
 
-    if (options.acceptLanguage !== null && options.acceptLanguage !== undefined) {
+    if(options.acceptLanguage !== null && options.acceptLanguage !== undefined) {
       this.acceptLanguage = options.acceptLanguage;
     }
-    if (options.longRunningOperationRetryTimeout !== null && options.longRunningOperationRetryTimeout !== undefined) {
+    if(options.longRunningOperationRetryTimeout !== null && options.longRunningOperationRetryTimeout !== undefined) {
```

Comment:
Is the editor formatting these files unintentionally?

Reply:
I did not open them in any editor. I think this is how the code generator generate code and probably the formatting hook was used before. Note that I did not push changes by the hook. I will see if there is a quick fix for this in the code generator.

Reply:
The iot central package was published only a month ago, so I doubt we changed anything in the code gen

Reply:
Yes I understand, I suspect that who did the generation last time also pushed the changes done by the formatting hook.

Reply:
I updated code gen to generate formatted code here.